src/compoenents/CustomerMailDetailPannel/IoTCard.tsx

```diff
@@ -1,17 +1,26 @@
-
 import React, { useEffect } from 'react';
 import '../../assets/styles/ioTCard.css';
 import { createChart } from './AmChartSetup'; // This will be a helper function for setting up the chart
 
 interface IoTCardProps {
   title: string;
-  data: { date: string, value: number }[];
+  data: { date: string, value: number }[]; // Define the correct structure for data
 }
 
 const IoTCard: React.FC<IoTCardProps> = ({ title, data }) => {
 
   useEffect(() => {
-    createChart(data, `${title}-chart`); // Create the chart using the dummy data
+    console.log(`useEffect for ${title}: Data length is`, data.length); // Log data length
```

Comment:
When we merge with dev branch keep in mind we need to remove these debugging console logs from code base